src/JustEat.StatsD/SupportedTagsFormatter.cs

```diff
@@ -0,0 +1,40 @@
+using JustEat.StatsD.Buffered.Tags;
+
+namespace JustEat.StatsD
+{
+    /// <summary>
+    /// Supported <see cref="IStatsDTagsFormatter"/> implementations for some of the major providers.
+    /// </summary>
+    public static class SupportedTagsFormatter
+    {
+        /// <summary>
+        /// Gets the AWS CloudWatch tags formatter.
```

Comment:
Changing _the_ to _a(n)_ as they're new objects on every access.

```suggestion
        /// Gets an AWS CloudWatch tags formatter.
```